dcrrates/server/sample-dcrrates-server.conf

```diff
@@ -0,0 +1,26 @@
+; The address and port on which to listen for gRPC connections.
+;listen=:7778
+
+; A directory to put log files. The directory will be created if it doesn't
+; exist.
+;logpath=logs
+
+; Disable individual exchanges. Multiple exchanges can be disabled with a
+; comma-separated list. Currently available: coinbase, coindesk, binance,
+; bittrex, dragonex, huobi, poloniex.
+;disable-exchange=
```

Comment:
does not seem to have an effect

Reply:
I wasn't able to reproduce this. Could you check again with the new commit?